bin/convert_dpc_folder.rb

```diff
@@ -47,7 +50,12 @@
   exit(false)
 end
 
-converter_args = [ options[:source], options[:target], options[:item_id_length], options[:checksums], options[:copy] ]
+converter_args = [ source: options[:source],
+                   target: options[:target],
+                   item_id_length: options[:item_id_length],
+                   checksum_file: options[:checksums],
```

Comment:
Is it important for the options keys to be different from the converter_args keys?